Dockerfile

```diff
@@ -25,11 +25,6 @@ ADD entrypoint.sh /usr/local/bin/
 ADD 50-config.py 60-database.sh 99-run.sh /startup/
 
 USER omero-server
-
-# Fixed in 5.5.0 https://github.com/openmicroscopy/openmicroscopy/pull/5949
-RUN sed -i.bak -re \
-    's/(name="omero.fs.importArgs"\s+value=)""/\1"--no-upgrade-check"/' \
-    /opt/omero/server/OMERO.server/etc/templates/grid/templates.xml
 EXPOSE 4063 4064
 VOLUME ["/OMERO", "/opt/omero/server/OMERO.server/var"]
```

Comment:
Not for this PR, but since we're makign breaking changes for Python 3 having `/opt/omero/server/OMERO.server/var` as a volume causes problems when you're building downstream images which modify OMERO.server, might be worth moving var out of `OMERO.server` and symlinking it. Either in Docker or in Ansible.